### ISSUE TITLE
Pause prop / control bug

### ISSUE THREAD

**Issue by fendorio** [itsnubix/react-native-video-controls#46]
I see there is an issue similar to this headed "...Android", I think that this is not specific to Android though so I have created a new issue, feel free to ask to migrate this over to the existing one. 

Basically, the play/pause button does not function correctly if you initialise the VideoPlayer component with paused being true. 

If you initialise the component with paused={true} then the video is indefinitely paused, i.e. pressing play does not play the video - the UI updates (play icon is swapped with pause), but the video itself remains in a paused state. 

In short, the pretty much renders the paused prop... not very useful? 

I have only tested this on IOS so far, will try test on Android before the week is out. 

Further detail and some code snippets below. 

IOS 10.3.2
Device: Iphone 5
React native: 0.47.2 (and 0.45). 

Code snippets below to demo what I have found: 

**Example 1**
If we intialize the component with paused={true}, then the play button does not work, you cannot use the controls to play the video, i.e. it is stuck in a "paused" state. 
 
```
<VideoPlayer
        paused={true} //Video starts as paused, pressing the play button does not start the video
/>
```

**Example 2**

Initializing the component with paused={false} does in-fact result in the play/pause button working. Although, this is problematic if you do not want the video to play automatically. 

```
<VideoPlayer
        paused={false} //Video starts playing, pressing the play button does play / pause the video
/>
```

**Comment by Pianist038801** [comment]
Exactly same thing happened to me.
In my last project, I used react-native-video@1.2.0 and react-native-video-controls@1.3.0 and everything was working fine.
But in the latest version :(

**Comment by kylemilloy** [comment]
Likely an issue because I'm mass assigning props now...will take a look at this.

**Comment by kylemilloy** [comment]
Resolved in v1.4.1
